sdk/messaging/azeventhubs/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.1.1 (Unreleased)
-
-### Features Added
+## 1.2.0 (2024-05-07)
 
 ### Breaking Changes
 
+- Processor.Run() behavior has been clarified and returns errors and handles closing
```

Comment:
Is this really a breaking change?

Reply:
I suppose not. I was worried that people might have been "depending" on the buggy behavior without realizing it.

Reply:
(removed)